decksite/sql/47.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS rule (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
+  archetype_id INTEGER NOT NULL,
+  FOREIGN KEY(archetype_id) REFERENCES archetype(id) ON DELETE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS rule_card (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
+  rule_id INTEGER NOT NULL,
+  card VARCHAR(100) NOT NULL,
```

Comment:
We use VARCHAR(190) for card name columns as that the maximum size of a varchar in the utf8mb4 encoding.

Our Market Research Shows That Players Like Really Long Card Names So We Made this Card to Have the Absolute Longest Card Name Ever Elemental is actually 142 chars, not that it's going to be defining any archetypes any time soon.